src/scenes/cyberball.ts

```diff
@@ -366,10 +375,10 @@ export class CyberballScene extends Phaser.Scene {
         if (this.settings.timeLimit > 0 && this.settings.displayTimeLimit)
             this.timeLimitText.setText(this.getTimeString());
 
-        if (this.settings.timeLimit > 0 && Date.now() - this.startTime > this.settings.timeLimit) {
-            this.postEvent('global-time-limit');
-            this.gameOver();
-        }
+        // if (this.settings.timeLimit > 0 && Date.now() - this.startTime > this.settings.timeLimit) {
+        //     this.postEvent('global-time-limit');
```

Comment:
Why did this get commented out?

Reply:
I just moved it to the beginning of the function and forgot to delete it here.